src/packagedcode/models.py

```diff
@@ -711,8 +711,34 @@ class PackageData(IdentifiablePackageData):
         repr=True,
     )
 
+    holder= String(
```

Comment:
It makes more sense to have this just after copyrights:
src/packagedcode/models.py#L612.

We would also need to regen the test files also.